atom/browser/atom_process_singleton.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2013 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ATOM_BROWSER_ATOM_PROCESS_SINGLETON_H_
+#define ATOM_BROWSER_ATOM_PROCESS_SINGLETON_H_
+
+#include "base/basictypes.h"
+#include "base/files/file_path.h"
+#include "chrome/browser/process_singleton.h"
+#include "chrome/browser/process_singleton_startup_lock.h"
+
+// Composes a basic ProcessSingleton with ProcessSingletonStartupLock
+class AtomProcessSingleton {
```

Comment:
This class seems redundant to me, we can probably just use `startup_lock_` and `process_singleton_` directly.